docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,141 @@
+# Component Manager
+
+Component manager provides information about installed and running components.
+
+The main features of the `Tizen.Applications.ComponentBased.ComponentManager` class include:
+
+-   Managing running component context
+
+    For the running components, you can retrieve the component running context and operate on it. You can [manage the component running context](#manage_context) with the `Tizen.Applications.ComponentBased.ComponentManager` class.
+
+-   Managing component information
+
+    For components that are installed but not necessarily running, you can retrieve information with the [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) class.
+
+Iterator methods are used to travel through a list of components. The `GetRunningComponentsAsync()` method of the `Tizen.Applications.ComponentBased.ComponentManager` class is used for the running components and the `GetInstalledComponentsAsync()` method is used for the installed components.
+
+## Prerequisites
+
+To enable your application to use the component management functionality:
+
+1.  To use classes and methods, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+       <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+    </privileges>
+    ```
+
+2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentManager.html), [Tizen.Applications.ComponentBased.ComponentRunningContext](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentRunningContext.html), and [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) classes, include the [Tizen.Applications.ComponentBased](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.html) namespace in your application:
+
+    ```
+    using Tizen.Applications.ComponentBased;
+    ```
+
+<a name="manage_context"></a>
+## Managing Running Component Context
+
+To get the component running context and its details, and to operate on the context:
```

Comment:
How about changing "component" and "running"?

To get the running component context and its details, and to operate on the context:

Reply:
@mijin85cho, thank you for the suggestion. I agree with it.

Reply:
Done